charts/gardener/provider-local/templates/deployment.yaml

```diff
@@ -32,6 +32,7 @@ spec:
         networking.gardener.cloud/to-runtime-apiserver: allowed
         networking.gardener.cloud/to-dns: allowed
         networking.gardener.cloud/to-kind-network: allowed
+        networking.gardener.cloud/to-virtual-garden-istio: allowed
```

Comment:
Use `networking.resources.gardener.cloud/to-all-istio-ingresses-istio-ingressgateway-tcp-9443: allowed` instead and add `{"matchLabels":{"gardener.cloud/role":"extension"}}` to https://github.com/gardener/gardener/blob/dcf0191b4f2a883f9115fd8316bc6884316abdf6/pkg/component/networking/istio/charts/istio/istio-ingress/templates/service.yaml#L8